src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,34 +2,42 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int EMPTY_STACK_VALUE = -1;
 
-	public int total = -1;
+	private static final int STACK_CAPACITY = 12;
 
-	public void countIn(int in) {
+	private final int[] numbers = new int[STACK_CAPACITY];
+
+	private int total = EMPTY_STACK_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)